scripts/get-downloader-release.js

```diff
@@ -1,34 +1,41 @@
 const https = require('https');
 const fs = require('fs');
-const url = require('url');
 
 const REPO = "eisopux/checker-framework-languageserver-downloader";
 const API_URL = `https://api.github.com/repos/${REPO}/releases/latest`;
 const JAR = "./checker-framework-languageserver-downloader.jar";
 
+function downloadFile(url, destination, callback) {
+    https.get(url, (response) => {
+        if (response.statusCode === 302) {
```

Comment:
Here is the redirect.

Reply:
Can you add a comment to the PR description what problem this is fixing?
Here, a comment about what response 302 means would also be useful.

Reply:
Already added.